test/e2e/Tests/Tests/PurgeInstancesTests.cs

```diff
@@ -23,27 +23,79 @@ public PurgeInstancesTests(FunctionAppFixture fixture, ITestOutputHelper testOut
         _output = testOutputHelper;
     }
 
-    // Due to some kind of asynchronous race condition in XUnit, when running these tests in pipelines,
-    // the output may be disposed before the message is written. Just ignore these types of errors for now. 
-    private void WriteOutput(string message)
+    [Fact]
+    public async Task PurgeOrchestrationHistory_StartAndEnd_Succeeds()
     {
-        try
-        {
-            _output.WriteLine(message);
-        }
-        catch
-        {
-            // Ignore
-        }
+        DateTime purgeStartTime = DateTime.MinValue;
+        DateTime purgeEndTime = DateTime.UtcNow;
+        string queryParams = $"?purgeStartTime={purgeStartTime.ToString("o")}&purgeEndTime={purgeEndTime.ToString("o")}";
```

Comment:
tip: this can be simplified further by removing the `ToString()` method call but leaving the format specifier:
```suggestion
        string queryParams = $"?purgeStartTime={purgeStartTime:o}&purgeEndTime={purgeEndTime:o}";
```

Reply:
Doen